### PR TITLE
ORLP-198.Create a method for updating cards on the CardController

### PR DESCRIPTION
**Pull request by IhorF**
Create a method for updating cards on the CardController. Change test data TestData.sql.

### CODE REVIEW

File: src/main/java/com/softserve/academy/spaced/repetition/service/CardService.java [hunk 1]
```diff
@@ -32,11 +32,13 @@ public Card getCard(Long id) {
 
     public void addCard(Card card, Long deckId) {
         Deck deck = deckRepository.findOne(deckId);
+        card.setDeck(deck);
         deck.getCards().add(cardRepository.save(card));
     }
 
     public void updateCard(Long id, Card card) {
         card.setId(id);
```
Comment (antonbakalets) [inline src/main/java/com/softserve/academy/spaced/repetition/service/CardService.java]: This method does nothing?
As for me, it reads from DB and saves the same into.

Comment (b0hdan) [reply src/main/java/com/softserve/academy/spaced/repetition/service/CardService.java]: I don't think so. It sets the id of the deck to the card, that is it adds that card to the deck.

Comment (b0hdan) [reply src/main/java/com/softserve/academy/spaced/repetition/service/CardService.java]: Oh, I'm sorry. I thought of the previous method earlier) You seem to be right.

Comment (b0hdan) [reply src/main/java/com/softserve/academy/spaced/repetition/service/CardService.java]: Oh, I'm sorry. I thought of the previous method earlier) You seem to be right.

Comment (b0hdan) [reply src/main/java/com/softserve/academy/spaced/repetition/service/CardService.java]: Is the problem only in the name?

**Pull request merged**